README.md

```diff
@@ -1,5 +1,154 @@
+    
+    # Port the app listens on. Required.
+    port: 8000
+    
+    # Optional. If the app is configured to listen on a base URL path, specify it here.
+    # If absent, we assume the app listens on / and we will set up an nginx reverse proxy based on
+    # the service name. However, not all apps are guaranteed to behave correctly under reverse proxies;
+    # so if possible, configure your app with a base URL.
+    baseUrl: "/jupyter/"
+    
+    # Optional. Specifies a startup command and arguments for the app. This overrides the Dockerfile
+    # ENTRYPOINT if present.
+    command:  ["/usr/local/bin/jupyter"]
+    args: ["notebook"]
+    
+    # Optional. Specifies the mount point for an attached PD and the access mode (ReadWriteOnce or ReadOnlyMany
+    # for read-write or read-only disks respectively). If absent no persistent volumes will be created.
+    pdMountPath: "/data"
+    pdAccessMode: "ReadWriteOnce"
+    
+    # Optional. Specifies environment variables to be present in the container.
+    environment:
+      WORKSPACE_NAME: "my-ws"
+      WORKSPACE_NAMESPACE: "my-proj"
+```
+
+# Supported Apps
+
+Example apps can be found in this repo under [apps](/apps). View the following for more details:
+
+[cellxgene](apps/cellxgene)
+
+[cirrocumulus](apps/cirrocumulus)
+
+[jupyter](apps/jupyter)
+
+[rstduio](apps/rstduio)
+
+[ucsc_genome_browser](apps/ucsc_genome_browser)
+
+# Launching an App Locally
+
+This repo contains a `terra-app-local.sh` script which can launch apps on a locally running [minikube](https://minikube.sigs.k8s.io/docs/) cluster. This can be much more convenient for development than launching apps against Terra.
+
+Before running the script:
+1. Install `minikube` according to instructions [here](https://minikube.sigs.k8s.io/docs/start/).
+2. Start `minikube` service.
+   a. On Mac (Catalina) I start it with the command:
+     ```
+     minikube start --vm=true --mount --mount-string="~/data:/data"
+     ```
+     Replace the data directory with the directory you want to mount into your persistent volume (or omit if not using persistence).
+    
+Once `minikube` is running, you can run the script. Here is the usage:
+```
+$ ./terra-app-local.sh 
+terra-app-local.sh deploys an app on a local minikube cluster.
+
+ Find more information at: https://github.com/DataBiosphere/terra-app
+
+Usage:
+  terra-app-local.sh [command]
+
+Available commands:
+  install       installs an app
+  uninstall     deletes an app
+  status        displays stats about an app
+  list          lists all apps
+  show-kubectl  prints kubectl commands to interact with an app
+
+Flags:
+  -h, --help    display help
+
+Use "terra-app-local.sh [command] --help" for more information about a command.
+```
+
+Example:
+```
+$ ./terra-app-local.sh install -f /apps/jupyter/app.yaml
+```
+
+# Launching an App on Terra
+ 
+TODO :)
+
+# Helm Chart
+
+The helm chart to deploy apps lives in this repo under [charts](charts). (Work in progress)
```

Comment:
Was the link to the charts supposed to be as below?
```suggestion
The helm chart to deploy apps lives in this repo under [charts](https://github.com/DataBiosphere/terra-app/tree/main/chart). (Work in progress)
```

Reply:
Ah yeah - I thought relative links worked but I guess not

Reply:
Ah maybe should be `[chart](chart)`